app/src/components/market/sections/market_list/market_home_container.tsx

```diff
@@ -59,6 +60,10 @@ const MarketHomeContainer: React.FC = () => {
   const [markets, setMarkets] = useState<RemoteData<MarketMakerDataItem[]>>(RemoteData.notAsked())
   const [cpkAddress, setCpkAddress] = useState<Maybe<string>>(null)
   const [moreMarkets, setMoreMarkets] = useState(true)
+  const calcNow = () => (Date.now() / 1000).toFixed(0)
```

Comment:
`useCallback`. Not really necessary here since this component is high in the tree, but still.